index.html

```diff
@@ -4,83 +4,219 @@
 <head>
 	<meta charset="utf-8" />
 	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
-
+<link rel="icon" href="./images/logo-circle.png">
 	<meta content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0' name='viewport' />
 	<meta name="viewport" content="width=device-width" />
-
-	<title>Virtision - XR Social App</title>
-
-	<link href="css/bootstrap.css" rel="stylesheet" />
+	<meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.virtision.com/" />
+    <meta property="og:title" content="Virtision - The First XR Social Music Platform" />
+    <meta
+      property="og:description"
+      content="Easily create, share and enjoy XR music worlds together all in one place. You can use any device or web browser to play and have fun"
+    />
+    <meta
+      property="og:image"
+      content="./images/meta.png"
+    />
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content="https://www.virtision.com/" />
+    <meta
+      property="twitter:title"
+      content="Virtision - The First XR Social Music Platform"
+    />
+    <meta
+      property="twitter:description"
+      content="Easily create, share and enjoy XR music worlds together all in one place. You can use any device or web browser to play and have fun"
+    />
+    <meta
+      property="twitter:image"
+      content="./images/meta.png"
+    />
+    <title>Virtision - The XR Social Music Platform</title>
+    <meta charset="utf-8" />
+    <meta name="title" content="Virtision - The First XR Social Music Platform" />
+    <meta
+      name="description"
+      content="Easily create, share and enjoy XR music worlds together all in one place. You can use any device or web browser to play and have fun"
+    />
+    <meta
+      name="keywords"
+      content="HTML, CSS, JavaScript, WebXR, A-frame, Three.JS, AR, VR, Mixed Reality, Creators, Influencers, Interactive, Music, Entertainment, Social Media, Music, Future, Metaverse, Creative, 3D Animation, Mobile Application"
+    />
+
+
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
 	<link href="css/main.css" rel="stylesheet" />
 
 	<!--     Fonts     -->
-	<link href="https://netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css" rel="stylesheet">
-	<script src="js/jquery-1.10.2.js" type="text/javascript"></script>
-	<script src="js/bootstrap.min.js" type="text/javascript"></script>
+	<link href="http://netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css" rel="stylesheet">
```

Comment:
need to use HTTPS for this one. original was https, so just use the old link

Reply:
okay